dyn_em/solve_em.F

```diff
@@ -4746,6 +4749,55 @@ SUBROUTINE solve_em ( grid , config_flags  &
    ENDIF
 #endif
 
+   
+!-----------------------------------------------------------------------
+! fire spotting (passive Lagrangian particle transport, tracks firebrand physics properties)
+!-----------------------------------------------------------------------
+! M. Frediani
+
+   IF(config_flags%ifire == 2 .AND. &
+       ! Check if spotting is on
+       config_flags%fs_firebrand_gen_lim > 0 .AND. & 
+       ! Check if this is the inner most grid
+       config_flags%max_dom == grid%id) THEN 
+
+       !BENCH_START(fire_driver_tim)
+
+       CALL wrf_debug ( 3 , 'solve: calling firebrand_spotting_em_driver...' )
+
+#      include "HALO_FIREBRAND_SPOTTING_3.inc"
+#      include "HALO_FIREBRAND_SPOTTING_5.inc"
```

Comment:
@mefrediani 
Put the two new HALO_FIREBRAND include files inside of this ifdef:
```
#ifdef DM_PARALLEL
#endif
```

That will fix the build errors for Serial and OpenMP.

Also, look about 20 lines above. It will show you how to incorporate some namelist options in to choosing the correct include file.